test/utils/DateFormatter.test.ts

```diff
@@ -0,0 +1,50 @@
+import { expect } from 'chai';
+
+import { dateToLocalTimeString } from 'app/utils/DateFormatter';
+
+describe('DateFormatter', () => {
+
+    it('should throw invalid date error', () =>{
+
+        const date: Date = new Date('nonsense date value');
+        expect(() => dateToLocalTimeString(date)).throws('DateFormatter - Invalid date');
+
+    });
+
+    it('should return YYYY-MM-DD format for a local date', () =>{
```

Comment:
What about scenario where timezones differ? The main purpose of creating that function is not really tested.